cli/tests/unit/webcrypto_test.ts

```diff
@@ -513,6 +519,31 @@ unitTest(async function testHkdfDeriveBits() {
   assertEquals(result.byteLength, 128 / 8);
 });
 
+unitTest(async function testHkdfDeriveBitsWithLargeKeySize() {
+  const key = await crypto.subtle.importKey(
+    "raw",
+    new Uint8Array([0x00]),
+    "HKDF",
+    false,
+    ["deriveBits"],
+  );
+  assertThrowsAsync(
```

Comment:
Prefer to use `assertRejects`

Reply:
Hi, 

Thank you for reviewing; please check new revision.

Thank you,
Upendra Upadhyay.